docs/Using Fleet/Vulnerability-Processing.md

```diff
@@ -399,15 +431,22 @@ The CPE translation. Used to match CPEs in the CPE database. Fields are are AND'
 
 | Name        | Type          | Description              |
 | ---         | ---           | ---                      |
-| `product`   | array[string] | The CPE product.         |
+| `product`   | array[string] | The CPE product.        |
 | `vendor`    | array[string] | The CPE vendor.          |
 | `target_sw` | array[string] | The CPE target software. |
 | `skip`      | bool          | If true, matched software will be skipped from the NVD vulnerability scanning process |
 
 ### Matching a CPE to a CVE
 
-Once we have a good CPE, we can match it against the CVE database. We download the data streams locally and match each CPE to the whole list. The matching is done using the [nvdtools implementation](https://github.com/facebookincubator/nvdtools).
+Once we have a good CPE, we can match it against the CVE database. We download the data streams
+locally and match each CPE to the whole list. The matching is done using the [nvdtools
+implementation](https://github.com/facebookincubator/nvdtools).
+
+### False Positive Cleanup
```

Comment:
[Sentence-case](https://fleetdm.com/handbook/company/communications#using-sentence-case-and-capitalization) heading:
```suggestion
### False positive cleanup
```